website/docs/quick-start-guide.md

```diff
@@ -1504,7 +1514,10 @@ spark.read.format("hudi"). \
 
 <TabItem value="sparksql">
 
```

Comment:
Updated to reflect current behavior which may be incorrect https://issues.apache.org/jira/browse/HUDI-6021